avm/res/automation/automation-account/main.bicep

```
@@ -194,6 +193,19 @@ resource automationAccount 'Microsoft.Automation/automationAccounts@2022-08-08'
   }
 }
 
+module automationAccount_credentials 'credential/main.bicep' = [
+  for (credential, index) in (credentials ?? []): {
+    name: '${uniqueString(deployment().name, location)}-AutomationAccount-Credential-${index}'
+    params: {
+      credentialName: credential.credentialName
+      credentialDescription: credential.credentialDescription
```

Unless you want to make it mandatory 😉 
```suggestion
      credentialDescription: credential.?description
```

---

updated